docs/UserGuide.md

```diff
@@ -20,16 +20,19 @@ It is optimized for a _Command Line Interface_ (CLI), but you can still use the
 
 ## Getting Started!
 
-> [!IMPORTANT]
+> [IMPORTANT]
```

Comment:
I think since the github markdown syntax might not work when converted to the website, maybe we can do it like
**IMPORTANT:** to make it bolded instead